packages/ui-contexts/src/ServerContext/methods.ts

```diff
@@ -20,6 +20,7 @@ export interface ServerMethods {
 	'updateIncomingIntegration': (...args: any[]) => any;
 	'updateOutgoingIntegration': (...args: any[]) => any;
 	'checkRegistrationSecretURL'(hash: string): boolean;
+	setUsername(username: string, param?: { joinDefaultChannelsSilenced?: boolean }): Promise<string>;
```

Comment:
oh, now that I get to the end of the page I see you "moved it back" to here.

from what I got the change to a augmentation is the expected as the method typing is not supposed to be spread outside the Meteor project.

Reply:
🤔 All right, can restore the agumentation. But, why are we doing the augmentation way? Like, we have around 12 methods here, what's the diff between the methods already here and the ones in the augmentation if all of them go to the same type? 👀